### PR TITLE
fix: add history entries for tars attribute

### PR DESCRIPTION
**Pull request by thesayyn**
image-spec says that this field is optional, however some systems like seen in https://github.com/bazel-contrib/rules_oci/issues/645 require this field. Also, skipping history entries lead to broken images if the base image already had one. 

fixes https://github.com/bazel-contrib/rules_oci/issues/645

### CODE REVIEW

**Comment by thesayyn** [comment]
This partially fixes https://github.com/bazel-contrib/rules_oci/issues/438 too.

File: oci/private/image.bzl [hunk 1]
```diff
@@ -105,6 +105,7 @@ def _calculate_descriptor(ctx, idx, layer, zstd, jq, coreutils, regctl):
     args = ctx.actions.args()
     args.add(layer)
     args.add(descriptor)
+    args.add(layer.owner)
```
Comment (alexeagle) [inline oci/private/image.bzl:108]: confused how this is related to the PR, is it just an unrelated drive-by fix?

**Review by alexeagle — APPROVED** [review]
